hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneClientAdapterImpl.java

```diff
@@ -407,6 +437,11 @@ public String getCanonicalServiceName() {
     return objectStore.getCanonicalServiceName();
   }
 
+  @VisibleForTesting
+  public void setClock(Clock monotonicClock) {
```

Comment:
I think we can reduce the visibility to default, as tests that should tinker with the clock can be - and for now they are - in the same package. What do you think?

Reply:
done